conf/messages

```diff
@@ -340,8 +340,7 @@ ssttp.direct-debit.guarantee.title=Direct Debit Guarantee
 ssttp.direct-debit.guarantee.list.1=This Guarantee is offered by all banks and building societies that accept instructions to pay Direct Debits.
 ssttp.direct-debit.guarantee.list.2=If there are any changes to the amount, date or frequency of your Direct Debit HM Revenue & Customs will notify you 10 working days in advance of your account being debited or as otherwise agreed. If you request HM Revenue & Customs to collect a payment, confirmation of the amount and date will be given to you at the time of the request.
 ssttp.direct-debit.guarantee.list.3=If an error is made in the payment of your Direct Debit by HM Revenue & Customs or your bank or building society you are entitled to a full and immediate refund of the amount paid from your bank or building society.
-ssttp.direct-debit.guarantee.list.4=If you receive a refund you are not entitled to, you must pay it back when HM Revenue & Customs asks you to.
-ssttp.direct-debit.guarantee.list.5=You can cancel a Direct Debit at any time by simply contacting your bank or building society. Written confirmation may be required. Please also notify us.
+ssttp.direct-debit.guarantee.list.4=If you receive a refund you are not entitled to, ssttp.arrangement.instalment-summary.printly contacting your bank or building society. Written confirmation may be required. Please also notify us.
```

Comment:
This line contains ssttp.arrangement.instalment-summary within the text message? Should the direct-debit guarantee even be changing? list.5 seems to have been deleted